clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/GoogleUserDataProvider.groovy

```diff
@@ -92,40 +101,45 @@ public class GoogleUserDataProvider {
    * %%detail%% 	    the detail component of the cluster name
    * %%launchconfig%% the name of the instance template
    */
-  private String replaceTokens(String rawUserData, BasicGoogleDeployDescription description,
+  private List<String> replaceTokens(List<String> rawUserData, BasicGoogleDeployDescription description,
                                String serverGroupName, String instanceTemplateName) {
     if (!rawUserData) {
-      return ''
+      return []
     }
     Names names = Names.parseName(serverGroupName)
     // Replace the tokens & return the result.
-    String result = rawUserData
-      .replace('%%account%%', description.accountName)
-      .replace('%%accounttype%%', description.credentials.accountType ?: '')
-      .replace('%%app%%', names.app ?: '')
-      .replace('%%env%%', description.credentials.environment ?: '')
-      .replace('%%region%%', description.region ?: '')
-      .replace('%%stack%%', description.stack ?: '')
-      .replace('%%group%%', names.group ?: '')
-      .replace('%%cluster%%', names.cluster ?: '')
-      .replace('%%detail%%', names.detail ?: '')
-      .replace('%%launchconfig%%', instanceTemplateName ?: '')
-
-    return result
+    return rawUserData.collect{userData ->
+      return userData.replace('%%account%%', description.accountName)
```

Comment:
I think it reads slightly better with all of the `.replace()`'s aligned.

Reply:
I can do that.